### PR TITLE
Add banner about WV2 support to Update README.md

### PR DESCRIPTION
**Pull request by lolambean**
Add banner about WV2 support

### CODE REVIEW

**Comment by lolambean** [comment]
#sign-off

Please review and merge this change.

**Comment by champnic** [comment]
Hey @lolambean thanks for the contribution. Are you with the Hololens team?

**Comment by champnic** [comment]
@lolambean I saw you were making edits to other Microsoft Docs - we probably need a similar change here: https://learn.microsoft.com/en-us/microsoft-edge/webview2/get-started/hololens2

Is that something you want to do, or should our team add similar verbage?

File: GettingStartedGuides/HoloLens2_GettingStarted/README.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ urlFragment: HoloLens2_GettingStarted
 ---
 # Get started with WebView2 in HoloLens 2 Unity apps
 
+> [!IMPORTANT]
+> WebView2 continues to be available. However, support for WebView2 is discontinued (no bug fixes, content updates or technical support), and we can therefore make no guarantees that applications will continue to work for any extended period of time.
```
Comment (champnic) [inline GettingStartedGuides/HoloLens2_GettingStarted/README.md:13]:
```suggestion
> WebView2 continues to be available. However, support for WebView2 on Hololens is discontinued (no bug fixes, content updates or technical support), and we can therefore make no guarantees that applications will continue to work for any extended period of time.

```

**Review by champnic — CHANGES_REQUESTED** [review]

File: GettingStartedGuides/HoloLens2_GettingStarted/README.md (hunk 1 above)
Comment (champnic) [reply GettingStartedGuides/HoloLens2_GettingStarted/README.md:13]: @lolambean requesting small change to make it more clear that this is only referring to support on Hololens.

Comment (lolambean) [reply GettingStartedGuides/HoloLens2_GettingStarted/README.md]: Perfect, that is this great suggestion.  I will make this change every where.

Comment (lolambean) [reply GettingStartedGuides/HoloLens2_GettingStarted/README.md]: @champnic I thought I accepted that change, but it doesn't look like it.  I am not sure what I am supposed to do next.  Please let me know.  This is a workflow I don't have a lot of experience with.

**Review by lolambean — COMMENTED** [review]
Added "on HoloLens".

**Review by champnic — APPROVED** [review]

**Comment by champnic** [comment]
@lolambean Merged - thanks again for the change! I've got the follow-up change to our other docs here:
https://github.com/MicrosoftDocs/edge-developer/pull/3317

**Comment by mikehoffms** [comment]
snapshot of draft PR 3317 - cleaned up wording but exactly same content as present PR:
image

**Pull request merged**